src/ShapeCrawler/Texts/TextVerticalAlignment.cs

```diff
@@ -0,0 +1,25 @@
+﻿// ReSharper disable once CheckNamespace
+
+namespace ShapeCrawler;
+
+/// <summary>
+///     Text Vertical alignment.
+/// </summary>
+public enum TextVerticalAlignment
```

Comment:
We already have an enum [TextAlignment](https://github.com/ShapeCrawler/ShapeCrawler/blob/master/src/ShapeCrawler/Texts/TextAlignment.cs) used for horizontal alignment of paragraph text. 

The new enum should be consistent with this. For example, you could rename the existing `TextAlignment` to `TextHorizontalAlignment`, or keep the single enum `TextAlignment` but rename/add new values like `HorizontalLeft`, `HorizontalCenter`, `VerticalTop`, etc.

Reply:
I think I will rename `TextAlignment` to `TextHorizontalAlignment` to avoid confusion. 

For the tests, it's going to be next week-end since I'll be away from my computer.